src/vs/code/electron-sandbox/issue/issueReporterPage.ts

```diff
@@ -16,6 +16,11 @@ export default (): string => `
 <div id="issue-reporter">
 	<div id="english" class="input-group hidden">${escape(localize('completeInEnglish', "Please complete the form in English."))}</div>
 
+	<div id="review-guidance-help-text" class="input-group">
+		${escape(localize('reviewGuidanceLabelText', "Before you report an issue here please {0}."))
+		.replace('{0}', `<a href="https://github.com/microsoft/vscode/wiki/Submitting-Bugs-and-Suggestions" target="_blank">${escape(localize('reviewGuidanceLinkText', "review the guidance we provide"))}</a>`)}
```

Comment:
You shouldn't need to do the `replace('{0}'`. The 3rd parameter on `localize` will do this replace automatically.

Reply:
I wrote it this way to try and avoid the risk of translators mangling the href text. The idea is they translate the hyperlinked text separately from the sentence in which the link appears. And some languages may need to move the hyperlinked text earlier in the sentence.

Reply:
The parameter values don't get sent to the translators though. So a translator sees only:

* `Before you report an issue here please {0}.`
and
* `review the guidance we provide`

This should really be one translation so that translators can correctly translate the full sentence.

Something like:

```

Before you report an issue here please {0} review the guidance we provide {1}.
```

And then have the next two parameters be the injected link.

Or maybe better for this case:

```
localize({ key: 'foo', comment:[
  '{Locked="<a href=\"....\">"}',
  '{Locked="</a>"
  ]},
  'Before you report an issue here please <a href="...">review the guidance we provide</a>.'
);
```

This uses a special comment syntax that programmatically locks the specified string so that translators can't mess it up.

Reply:
Thanks for the information. I have pushed a change for your review.